src/api/utils/MessageEmbedOptionsShape.ts

```diff
@@ -22,4 +23,6 @@ export const MessageEmbedOptionsShape: ZodShape<MessageEmbedOptions> = {
   image: z.any().optional(),
   video: z.any().optional(),
   footer: z.any().optional(),
+  provider: z.any().optional(),
```

Comment:
[`EmbedData` now also expects an optional value for `provider`.](https://github.com/discordjs/discord.js/blob/5bfcf93e29ab734da670d9955f38163268d6a675/packages/discord.js/typings/index.d.ts#L832)